docs/snap-clone.md

```diff
@@ -172,15 +172,15 @@ Name:         rbd-pvc-snapshot
 Namespace:    default
 Labels:       <none>
 Annotations:  <none>
-API Version:  snapshot.storage.k8s.io/v1alpha1
+API Version:  snapshot.storage.k8s.io/v1
```

Comment:
@humblec can you also update the describe and get outputs as they are added when cephcsi was using v1alpha1 API.?

Reply:
@Madhu-1 I didnt get above comment, can you explain a bit more on above ?

Reply:
the output of `kubectl get volumesnapshot` and `kubectl describe volumesnapshot rbd-pvc-snapshot` are from the v1alpha1 CRD. I think it got changed in V1 can you please check.

Reply:
@Madhu-1 updated ptal.. thanks.